src/Microsoft.ML.Data/Model/Onnx/ICanSaveOnnx.cs

```diff
@@ -30,6 +30,13 @@ public interface ITransformCanSaveOnnx: ICanSaveOnnx, IDataTransform
         void SaveAsOnnx(OnnxContext ctx);
     }
 
+    /// <summary>
+    /// This data model component is savable as ONNX.
+    /// </summary>
+    public interface ITransformCanSaveOnnx : ICanSaveOnnx, ISaveAsOnnx, IDataTransform
```

Comment:
>ICanSaveOnnx, ISaveAsOnnx [](start = 45, length = 25)

this is confusing: why are there two interfaces?

Reply:
I see a reason, but it doesn't make it less confusing to be honest

---
In reply to: [214106801](https://github.com/dotnet/machinelearning/pull/759#discussion_r214106801) [](ancestors = 214106801)

Reply:
I agree, but I don't want this to be scope of this PR.

---
In reply to: [214107108](https://github.com/dotnet/machinelearning/pull/759#discussion_r214107108) [](ancestors = 214107108,214106801)